pos/app/routes/_header.master.tsx

```diff
@@ -84,14 +108,15 @@ export default function FielsOfMaster() {
                         {order.comments.map((comment, index) => (
                           <div
                             key={`${comment.author}-${comment.text}`}
-                            className="my-2 flex rounded-md bg-gray-200 p-1"
+                            className="my-2 flex rounded-md bg-gray-200 px-2 py-1"
                           >
                             <div className="flex-none">{comment.author}：</div>
                             <div>{comment.text}</div>
                           </div>
                         ))}
                       </div>
                     )}
+                    <InputComment order={order} mutateOrder={mutateOrder} />
```

Comment:
serve の方も同様に直してみて下さい
```suggestion
                    <InputComment addCommentToOrder={useCallback((comment: string) => {
                      const commentAddedOrder = order.clone();
                      commentAddedOrder.addComment(comment);
                      mutateOrder(commentAddedOrder);
                      }, [/* この中は良い感じに */])}
                    />
```